mozilla_schema_generator/probes.py

```diff
@@ -153,15 +158,24 @@ def __init__(self, identifier: str, definition: dict, *, pings: List[str] = None
         super().__init__(identifier, definition)
 
         defn_pings = set(
-            [p for d in definition[self.history_key] for p in d.get("send_in_pings", ["metrics"])]
+            [
```

Comment:
Did you run black on this? That's fine by me, but in that case it would be good to add it to CI as well.

Reply:
I accidentally did, however haven't done so for any other files. I'd say adding black to CI might be best in a followup PR since there'd be a lot of changes.

Reply:
I added a comment in the code. Basically, before `_get` threw exceptions for accessing non-existing fields